.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Pull request
+about: Create a pull request for an issue in Altinn Studio
+---
+
+Thank you for contributing to Altinn:) We know this isn't the fun part, 
+but please put the same effort into the pull request as you did into the code and it should soon find it's way to master.
+
```

Comment:
In the meeting we had, we also discussed having some useful links here, f.ex link directly to the contribution guide (https://github.com/Altinn/altinn-studio/blob/master/CONTRIBUTING.md).

I know its a bit "late" in the process to link to this when opening a PR, but at the same time it will be very hard to miss it :)

Reply:
Yep. I thought of it when I put up the proposal, what caused me to not include it was 1) The fact that is late and 2) It quickly becomes something you ignore and just put a checkmark on and 3) For the same reason as bullet point 2 we should keep this as short and to the point as possible so that people actually consider them each time. But this was a borderline issue so happy to include it if we feel it makes a difference.

Reply:
Good point to have it as short as possible. 

I like the checklist at the end, that should probably be enough.